spec/system/candidate_interface/candidate_expire_session_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'rails_helper'
+
+describe 'Candidate session expires' do
+  it 'can expire the current candidate session after 30 minutes' do
+    candidate = FactoryBot.create(:candidate)
+    login_as(candidate)
+
+    visit candidate_interface_welcome_path
+
+    Timecop.travel(Time.now + 30.minutes + 1.second) do
+      visit candidate_interface_welcome_path
+
+      expect(page).to have_current_path(candidate_interface_start_path)
```

Comment:
should not we have also feedback about the expiration of the session?
I imagine the user might my surprised of the sudden redirection without a clear message.

Reply:
We're missing a few of these notices in our app but I think we can card them up later. The current plan is to get to a working end to end application journey first, and then refine everything at the end before launch. We can launch without this, though it's not ideal, but we can...

Reply:
I can have a look into if devise has some kind of notifications for this. 
Could be pretty simple to add (famous last words).

Reply:
Devise does provide a `notice` when redirecting on sign in and sign out, but not when timing out a session. 
I'd be fine adding notifications in a future PR :)